addons/binding/org.openhab.binding.knx/src/main/java/org/openhab/binding/knx/internal/channel/WriteSpecImpl.java

```diff
@@ -26,12 +28,13 @@
     private final Type type;
     private final @Nullable GroupAddress groupAddress;
 
+    private final Logger logger = LoggerFactory.getLogger(WriteSpecImpl.class);
```

Comment:
what do you need this one for?

Reply:
Debugging artefact. It should be removed indeed